squad/api/rest.py

```diff
@@ -762,15 +762,24 @@ class EnvironmentViewSet(ModelViewSet):
     ordering_fields = ('id', 'slug', 'name')
 
 
+class HyperlinkedMetricIdentityField(serializers.HyperlinkedIdentityField):
+    def get_url(self, *args):
+        testrun = args[0]
+        if testrun.metrics.count() == 0:
```

Comment:
it seems that this will produce one count(*) query for every test run in a list ...